src/lib/expression/evaluation/expression_result_views.hpp

```diff
@@ -86,14 +104,26 @@ class ExpressionResultLiteral {
 
   ExpressionResultLiteral(const T& value, const bool null) : _value(value), _null(null) {}
 
-  bool is_series() const { return false; }
-  bool is_literal() const { return true; }
-  bool is_nullable() const { return _null; }
+  bool is_series() const {
+    return false;
+  }
+  bool is_literal() const {
+    return true;
+  }
+  bool is_nullable() const {
+    return _null;
+  }
 
-  size_t size() const { return 1u; }
+  size_t size() const {
+    return 1u;
+  }
 
-  const T& value(const size_t /*value*/) const { return _value; }
-  bool is_null(const size_t /*value*/) const { return _null; }
+  const T& value(const size_t /*value*/) const {
```

Comment:
What was our POV on unused parameters again?

Reply:
This. :)

Use comments instead of leaving out the variable name. When applicable, use the same variable name as in similar functions. Otherwise, use /*unused*/.

Further I would add to line 82:
```
* Unused parameters: instead of leaving out the parameter name, comment the parameter name out (e.g.,
      `void function(const size_t /*unused*/) { ... }`).
 ```